contributors.md

```diff
@@ -225,9 +266,12 @@ I am an associate member of the Chartered Institute of Personnel and Development
 > If we say that we want people to make their research open and reproducible, we need to give them the tools they need to be confident in doing so.
 > I see the Turing Way as the means of bridging that gap, by providing a friendly, practical and helpful guide for researchers at all stages of their careers.
 
+<!---M--->
+
 
-### Malvika Sharan (Community Manager)
+### Malvika Sharan
 
+* Role: **Community Manager**
```

Comment:
Love that you made yourself bold 😉 
```suggestion
* Role: **Community Manager** (2019 - )
```

Reply:
haha - so that people can find me :D I left you on the top but moved my bio down as per alphabetical order to make sure I do what I preach.